e2e/pages/adf/permissionsPage.ts

```diff
@@ -92,4 +94,26 @@ export class PermissionsPage {
         return new DataTablePage(element(by.css('[class*="adf-datatable-permission"]')));
     }
 
+    checkUserHasRoleSelected(user, role) {
+        return new DataTablePage().getRowByRowNumber(user).element(by.xpath(`ancestor::div/div[contains(@class, 'adf-data-table-cell')]` +
```

Comment:
Please move the locator related to adf-data-table-cell to datatable page and also make sure you are using a general one in there.